WcaOnRails/app/assets/stylesheets/users.scss

```diff
@@ -20,6 +20,13 @@ td.unconfirmed-email {
   font-style: italic;
 }
 
+.pending-id {
+  background-color: #f2dede;
+  border-color: #ebccd1;
+  color: #a94442;
```

Comment:
These should go in the variables file (maybe you can even reuse existing colours).

Reply:
Actually this was derived from the alert-danger. But that style makes the text too big, IIRC.